src/main/kotlin/org/usfirst/frc/team4099/robot/Robot.kt

```diff
@@ -26,7 +33,17 @@ class Robot : IterativeRobot() {
 
     override fun robotInit() {
         try {
+            CameraServer.getInstance().startAutomaticCapture()
```

Comment:
Is this from last year? Doesn't really belong on this branch

Reply:
Yes it is from last year.